### PR TITLE
add threshold adjustment description for soil monisture sensor, fix #1719

### LINKED ISSUE #1719: [Bug Report]: 土壤湿度传感器描述不清楚
### 联系方式

_No response_

### 您发现了什么问题

https://haas.iot.aliyun.com/solution/detail/hardware?versionId=800CCF5AD5CBBCEA00000001&dataId=800CCF5AD5CBBCEA&s=
下图中的阈值是指什么？怎样调节阈值呢？文档中没有对怎样调节阈值进行描述。
image

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 8b936c21b52c80ce37ef1050726687dfad6c0e09

**Comment by skylarCai** [comment]
auto build success,number:221

**Comment by skylarCai** [comment]
autotest fail,number:1584

**Comment by skylarCai** [comment]
autotest fail,number:1585

**Review by wushengtws — APPROVED** [review]

File: haas_lib_bundles/python/libraries/soil_moisture/README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@
 <img src="./../../docs/images/soil_moisture_front.jpg"  width=40%/>
```
Comment (skylarCai) [inline haas_lib_bundles/python/libraries/soil_moisture/README.md:7]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**